ui/app/mainui/AppMain.qml

```diff
@@ -1174,30 +1176,20 @@ Item {
 
     Connections {
         target: appMain.rootStore.mainModuleInst
+
         function onActiveSectionChanged() {
-            let communitySectionModule = appMain.rootStore.mainModuleInst.getCommunitySectionModule()
-            if (communitySectionModule)
-                rootDropAreaPanel.activeChatType = communitySectionModule.activeItem.type
+            let sectionType = appMain.rootStore.mainModuleInst.activeSection.sectionType
+            rootDropAreaPanel.enabled = !rootDropAreaPanel.drag.source && (sectionType === Constants.appSection.chat ||
```

Comment:
This whole `Connections` now seems redundant; you already have `rootDropAreaPanel.enabled` as a binding below

Reply:
So what we can do is just remove the binding and keep that connections. WDYT?

Reply:
I'd rather keep the bindings, and let the DropArea decide based on `appMain.rootStore.mainModuleInst.activeSection.sectionType`

Reply:
Unfortunately, we don't have full control with `appMain.rootStore.mainModuleInst.activeSection.sectionType`. There is a specific case where the `sectionType` is community but the view is not a chat related one, they are the settings views. Here in settings we need to disable the main drag/drop area bc there are particular components that will need to drag/drop inside. Example: 

![Screenshot 2023-02-23 at 19 48 14](https://user-images.githubusercontent.com/97019400/221221956-aa4127e5-dfa6-49e8-87a0-3d2ab413d341.png)

So then, we need to force a manual assignement (the one done in `ChatLayout.qml`) when community section is active BUT seetings are displayed.

Reply:
Still doable with the binding imo; just need to expose current section (`currentIndex`) from ChatLayout for reading

Reply:
:thinking: ok.. I'll think on that! Good idea!

Reply:
I agree with @caybro. Would be nice to remove the js functions and keep bindings only.
I see the `DropAreaPanel` is exposed as `Global.dragArea`. WDYT if every component that should support Drag&Drop to create a [Binding](https://doc.qt.io/qt-5/qml-qtqml-binding.html) with restoreMode `Binding.RestoreBindingOrValue` on `Global.dragArea`?

There are a few advantages. It's easy to enable Drag/Drop on other views because and we avoid js logic cluttering in `AppMain.qml`.

Reply:
I really like your suggestion! Thanks!

I'll create the binding in the component instantiation instead of in the object definition so we can avoid using the `dragArea` as a global property in this case!

I've pushed updates, please @caybro @alexjba review them again!